src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -269,11 +289,22 @@ public enum BuildChainCustomTrustStoreTestArguments : int
             MultipleCalls
         }
 
+        public static IEnumerable<object[]> BuildChainCustomTrustStoreData()
+        {
+            if (!PlatformDetection.IsAndroid)
+            {
+                // Android doesn't support an empty custom root
+                yield return new object[] { false, X509ChainStatusFlags.UntrustedRoot, BuildChainCustomTrustStoreTestArguments.TrustedIntermediateUntrustedRoot };
+            }
+
+            yield return new object[] { true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.UntrustedIntermediateTrustedRoot };
+            yield return new object[] { true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.TrustedIntermediateTrustedRoot };
+            yield return new object[] { true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.MultipleCalls };
+        }
+
         [Theory]
-        [InlineData(false, X509ChainStatusFlags.UntrustedRoot, BuildChainCustomTrustStoreTestArguments.TrustedIntermediateUntrustedRoot)]
-        [InlineData(true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.UntrustedIntermediateTrustedRoot)]
-        [InlineData(true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.TrustedIntermediateTrustedRoot)]
-        [InlineData(true, X509ChainStatusFlags.NoError, BuildChainCustomTrustStoreTestArguments.MultipleCalls)]
+        [PlatformSpecific(~TestPlatforms.Android)] // Test relies on AIA fetching, which Android does not support
```

Comment:
The AIA fetch and system retrieval aren't really important to this test.  Making all platforms bring the data in explicitly is fine.

IIRC, it was just written before we bothered having all of the certs in this chain explicitly... so it was "eh, we already have a way of getting at this data".